src/test_l1_dsFPD.c

```diff
@@ -2836,21 +3041,23 @@ void test_l1_dsFPD_negative_dsFPSetLEDState(void)
  * |06|Compare the returned values from steps 3/4 ||Success|The values should be the same|
  * |07|Terminate using dsFPTerm()||dsERR_NONE|Ensure the system is terminated|
  * 
- * @note Valid indicators can retrieved from id element in kIndicators in the dsFPDSettings.h file
+ * @note Valid indicators can retrieved from yaml file
  */
 void test_l1_dsFPD_positive_dsFPGetLEDState(void)
 {
     gTestID = 41;
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
     dsError_t result;
     dsFPDLedState_t ledState1, ledState2;
+    int count=0;
 
     // Step 01: Initialize using dsFPInit()
     result = dsFPInit();
     UT_ASSERT_EQUAL(result, dsERR_NONE);
 
     // Step 02: Set all valid indicators to dsFPD_STATE_ON using dsSetFPState()
-    enableFPDIndicators();
+    count = UT_KVP_PROFILE_GET_UINT32("dsFPD/Number_of_Indicators");
```

Comment:
As SupportedLEDStates is also defined in profile, this should be used to validate.

Reply:
Done

Reply:
Please consider the SupportedLEDStates to validate the led state as well as this is now available in the test profile.